packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss.vue

```diff
@@ -27,7 +27,8 @@
 											{{ id }}
 										</td>
 										<td>
-											<tera-input-number
+											<tera-input
+												type="decimal"
```

Comment:
This isn't a valid type is it @blanchco ?

Reply:
Decimal seemed to work in a very quick test

Reply:
inputmode?: 'none' | 'text' | 'tel' | 'url' | 'email' | 'numeric' | 'decimal' | 'search';

Reply:
decimal should be fine...we may push the text to the right like we do with number though